### PR TITLE
Add oh my zsh installation instructions.

### PR DESCRIPTION
**Pull request by mathieutu**

### CODE REVIEW

**Comment by mathieutu** [comment]
Actually don't merge that right now, theme doesn't work when nested in a folder. I'll make a fix.

@kolo How did you handled that when you made #22? 
Because for now a simlink in the same folder seems useless.

We should add a `ln -s ${ZSH_CUSTOM}/themes/minimal/minimal.zsh ${ZSH_CUSTOM}/themes/minimal.zsh-theme` command after git clone.

**Comment by subnixr** [comment]
What's the status on this?

**Comment by mathieutu** [comment]
I was waiting for @kolo answer but nothing, so I'll update it and ping you when it's done.

File: README.md [hunk 1]
```diff
@@ -6,15 +6,22 @@ A minimal and extensible zsh theme.
...
 
 [Antigen](https://github.com/zsh-users/antigen):
-```
+```sh
 antigen bundle subnixr/minimal
 ```
 
+[Oh My Zsh](https://ohmyz.sh/):
+```sh
+git clone https://github.com/subnixr/minimal.git  ~/.oh-my-zsh/custom/themes/minimal
```
Comment (mathieutu) [inline README.md]:
```suggestion
git clone https://github.com/subnixr/minimal.git  ${ZSH_CUSTOM}/themes/minimal

ln -s ${ZSH_CUSTOM}/themes/minimal/minimal.zsh ${ZSH_CUSTOM}/themes/minimal.zsh-theme
```

Comment (mathieutu) [reply README.md:20]: Add symbolic link to root theme folder for omzsh.

**Comment by mathieutu** [comment]
@subnixr you can merge, thanks.

**Pull request merged**